### ISSUE TITLE
Add `meta` how-to guide

### ISSUE THREAD

**Issue by stevepiercy** [plone/documentation#1827]
Create a new page under [Developer guide](https://6.docs.plone.org/developer-guide/index.html). It will consist of an introduction of what `meta` is, with minimal usage docs for how to get started and links to details for routine tasks.

@gforcada fyi.

_Originally posted by @stevepiercy in https://github.com/plone/plone.api/issues/546#issuecomment-2558322208_

**Comment by gforcada** [comment]
Yes, that would be a very nice addition. Nice timing, as I'm in the process of turning `plone/meta` repository into a python distribution that can be installed. I will try to do a minimal proposal and see how we expand from there.
